tests/ha/check_after_reboot.pm

```diff
@@ -36,6 +36,12 @@ sub run {
     # in that case
     select_console 'root-console' if (get_var('HDDVERSION'));
 
+    # Workaround network timeout issue during upgrade
+    if (get_var('HDDVERSION')) {
+        systemctl 'restart iscsi';
+        systemctl 'restart pacemaker';
```

Comment:
You forget to add `use utils 'systemctl';`.

Reply:
Not anymore.